resources/views/product/partials/options/file.blade.php

```diff
@@ -1,13 +1,12 @@
 @if (Rapidez::checkcompadreVersion('0.0.1'))
-    <label>
-        <x-rapidez::label>{{ $option->title }} {{ $option->price_label }}</x-rapidez::label>
-        <x-rapidez::input
-            type="file"
-            :required="$option->is_require"
-            v-on:change="addToCart.setCustomOptionFile($event, {{ $option->option_id }})"
-            class="px-0"
-        />
-    </label>
+    <x-rapidez::input
+        type="file"
+        :required="$option->is_require"
+        v-on:change="addToCart.setCustomOptionFile($event, {{ $option->option_id }})"
+        class="px-0"
+    >
+        {{ $option->title }} {{ $option->price_label }}
```

Comment:
is this correct? shouldn't this be the label? now it's underneath your input as the slot is at the bottom